### DISCUSSION TITLE
Fe的warn日志疯狂打failed add to meta. tablet[1419031], backend[10002]. errCode = 7, detailMessage = tablet meta[null] does not exist in tablet inverted index日志

### DISCUSSION THREAD

**Discussion by young138120** [apache/doris#14998] [A - General / Q&A]
Doris版本：1.0.0-rc03
Fe节点定时疯狂的输出如上的日志，看了下fe的代码，应该是定时生成的TReportRequest任务中backendTablets和执行任务中获取的foundTabletsWithValidSchema不匹配，然后去进行删除操作
什么情况下会出现这种情况，以及我怎么处理这种情况

**Comment by yuanyuan8983** [comment]
It is recommended to try upgrading to version 1.1.5. This problem is probably a known bug.

**Comment by lymaimei** [comment]
> Doris版本：1.0.0-rc03 Fe节点定时疯狂的输出如上的日志，看了下fe的代码，应该是定时生成的TReportRequest任务中backendTablets和执行任务中获取的foundTabletsWithValidSchema不匹配，然后去进行删除操作 什么情况下会出现这种情况，以及我怎么处理这种情况

类似日志在1.2.1中出现，delete操作提示
failed to execute delete. transaction id 357365, timeout(ms) 30000, unfinished replicas [BackendId=TabletId]:
查询正常 
求解决办法
